extensions/arc/deployment/src/main/resources/dev-templates/beanDependencyGraph.html

```diff
@@ -137,22 +162,47 @@
         </div>
     </div>
     <div class="row">
-        <div class="col-md-auto">
+        <div class="col-2">
             <ul class="list-group list-group-flush">
                 <li class="list-group-item"><i class="fas fa-circle text-danger fa-sm legend-root"></i> root</li>
                 <li class="list-group-item"><i class="fa fa-long-arrow-right fa-sm legend-direct-dependency"></i> direct dependencies</li>
                 <li class="list-group-item"><i class="fa fa-long-arrow-right fa-sm legend-direct-dependent"></i> direct dependents</li>
                 <li class="list-group-item"><i class="fa fa-long-arrow-right fa-sm legend-dependency"></i> dependencies</li>
                 <li class="list-group-item"><i class="fa fa-long-arrow-right fa-sm legend-producer"></i> declaring bean of a producer</li>
                 <li class="list-group-item"><i class="fa fa-long-arrow-right fa-sm legend-lookup"></i> potential dependency <br/> &nbsp;<small class="text-muted">programmatic lookup</small></li>
+                <li class="list-group-item">
+                    <button class="btn btn-primary" type="button" id="zoom_in">
+                        <i class="fas fa-search-plus"></i> Zoom in
+                    </button>
+                    <button class="btn btn-primary" type="button" id="zoom_out">
+                        <i class="fas fa-search-minus"></i> Zoom out
+                    </button>
```

Comment:
```suggestion
<div class="btn-group" role="group" aria-label="Zoom">
    <button class="btn btn-light" type="button" id="zoom_in">
        Zoom in <i class="fas fa-search-plus"></i>
    </button>
    <button class="btn btn-light" type="button" id="zoom_out">
        <i class="fas fa-search-minus"></i> Zoom out
    </button>
</div>
```

Can you see how the above looks like ? Not sure if it will be better, but the primary button feels to heavy.

Reply:
I find the `btn-light` buttons too light, barely visible but `btn-secondary` looks better IMO...